### PR TITLE
MODAUD-20 - Check In/Check Out: log actions

### PR DESCRIPTION
**Pull request by khandramai**
[MODAUD-20](https://issues.folio.org/browse/MODAUD-20) - Check In/Check Out: log actions

### CODE REVIEW

File: mod-audit-server/src/main/java/org/folio/builder/LogRecordBuilderResolver.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  public static final String CHECK_IN_EVENT = "CHECK_IN_EVENT";
+  public static final String CHECK_OUT_EVENT = "CHECK_OUT_EVENT";
+
+  private LogRecordBuilderResolver() { }
+
+  public static LogRecordBuilder getBuilder(String logEventType) {
+    if (logEventType.equals(CHECK_IN_EVENT)) {
+      return new CheckInRecordBuilder();
+    } else if (logEventType.equals(CHECK_OUT_EVENT)) {
+      return new CheckInRecordBuilder();
```
Comment (siarhei-charniak) [inline mod-audit-server/src/main/java/org/folio/builder/LogRecordBuilderResolver.java:17]: Should `CheckInRecordBuilder` be created in both cases?

**Review by siarhei-charniak — APPROVED** [review]

File: mod-audit-server/src/main/java/org/folio/util/JsonPropertyFetcher.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package org.folio.util;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.joda.time.DateTime.parse;
+
+import org.joda.time.DateTime;
```
Comment (grabsefx) [inline mod-audit-server/src/main/java/org/folio/util/JsonPropertyFetcher.java:6]: My suggestion would be to get rid of obsolete `org.joda.time` library and use `LocalDateTime` available out of the box

Comment (khandramai) [reply mod-audit-server/src/main/java/org/folio/util/JsonPropertyFetcher.java:6]: This is unified approach that used in the mod-circulation. Could you please confirm that soda DateTime -> Json date/time -> LocalDate time will work as expected?

Comment (grabsefx) [reply mod-audit-server/src/main/java/org/folio/util/JsonPropertyFetcher.java:6]: I see. There is no other simple ways to keep date compatible with mod-circulation format

**Pull request merged**